src/service/fixed-layer.js

```diff
@@ -988,7 +989,7 @@ class TransferLayerDef {
 class TransferLayerBody {
   /**
    * @param {!Document} doc
-   * @param {!./vsync-impl.Vsync} vsync
+   * @param {!./vsync-impl.Vsync}  vsync
```

Comment:
intentional?

Reply:
I'm not sure why that happened, but I'll fix it.